entgql/template/pagination.tmpl

```diff
@@ -434,13 +449,18 @@ func (p *{{ $pager }}) orderExpr(query *{{ $node.QueryName }}) sql.Querier {
 		})
 	{{- else }}
 		direction := p.order.Direction
+		nullsDirection := p.order.NullsDirection
+		if nullsDirection == "" {
```

Comment:
Same concern here with the difference in how `direction` and `nullsDirection` is being handled here.